python_scripts/metrics.py

```diff
@@ -634,8 +646,9 @@ def plot_pr_roc_interactive():
 
 # %% [markdown]
 # The :math:`R^2` score represents the proportion of variance of the target
-# explained by the independent variables in the model. The best score possible
-# is 1 but there is no lower bound. However, a model which would predict the
+# that is explained by the independent variables in the model. The best score
+# possible
+# is 1 but there is no lower bound. However, a model that predicts the
```

Comment:
We could expand on this. 0 is no variation in target explained (always predict mean target value value) and less than 0 - your model performs worse than just predicting the mean.